app-emulation/libdsk/libdsk-1.5.8.recipe

```diff
@@ -12,66 +12,72 @@ controller emulation, thus giving the emulator transparent access to .DSK \
 files or real discs.
 "
 HOMEPAGE="http://www.seasip.info/Unix/LibDsk/"
-COPYRIGHT="2010-2015 John Elliott"
+COPYRIGHT="2010-2018 John Elliott"
 LICENSE="GNU GPL v2"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="http://www.seasip.info/Unix/LibDsk/libdsk-$portVersion.tar.gz"
-CHECKSUM_SHA256="e1c56f6395a011d508a5e548dcbcadce8ee8cb95740f986d6e8e3695ed59de82"
+CHECKSUM_SHA256="0d3b4dbe1986682bd38fa3cb50aa6f37190f8adac32e0b9f88644df9ccbcbeb9"
 
-ARCHITECTURES="x86 x86_gcc2"
-SECONDARY_ARCHITECTURES="x86 x86_gcc2"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+libVersion="4.3.1"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
 PROVIDES="
 	libdsk$secondaryArchSuffix = $portVersion
-	lib:libdsk$secondaryArchSuffix = 4.3.0 compat >= 4
+	lib:libdsk$secondaryArchSuffix = $libVersionCompat
 	"
+if [ -z "$secondaryArchSuffix" ]; then
+	PROVIDES_tools="$PROVIDES
```

Comment:
```diff
 if [ -z "$secondaryArchSuffix" ]; then
- 	PROVIDES_tools="$PROVIDES
+	PROVIDES_tools="
+		libdsk${secondaryArchSuffix}_tools = $portVersion
		cmd:apriboot
		cmd:dskconv
```

Reply:
I just realised that we can (and probably should) use the following variant, instead of my first suggestion above:
```diff
 if [ -z "$secondaryArchSuffix" ]; then
- 	PROVIDES_tools="$PROVIDES
+	PROVIDES_tools="
+		libdsk_tools = $portVersion
		cmd:apriboot
		cmd:dskconv
```
(Indeed, we can drop the `${secondaryArchSuffix}` because `PROVIDES_tools` is only being defined if `$secondaryArchSuffix` is empty.)